play/src/front/Connexion/RoomConnection.ts

```diff
@@ -535,7 +536,11 @@ export class RoomConnection implements RoomConnection {
                 }
                 case "refreshRoomMessage": {
                     console.info("roomConnection => refreshRoomMessage received");
-                    window.location.reload();
+                    if (message.refreshRoomMessage.comment) {
+                        mapUploadRefreshNeededCommentStore.set(message.refreshRoomMessage.comment);
+                    } else {
+                        window.location.reload();
+                    }
```

Comment:
Could we move this out of `RoomConnection` class.
Maybe use a stream (just like with the other messages) and subscribe to that stream in the GameScene?

This way, we keep the `RoomConnection` class more agnostic.
I want that because I might reuse the `RoomConnection` in a performance test in the future and I need to be able to call it without triggering anything specific from the UI